zerver/views/__init__.py

```diff
@@ -1128,9 +1131,25 @@ def update_realm(request, user_profile, name=REQ(validator=check_string, default
                  restricted_to_domain=REQ(validator=check_bool, default=None),
                  invite_required=REQ(validator=check_bool, default=None),
                  invite_by_admins_only=REQ(validator=check_bool, default=None),
-                 create_stream_by_admins_only=REQ(validator=check_bool, default=None)):
-    # type: (HttpRequest, UserProfile, Optional[str], Optional[bool], Optional[bool], Optional[bool], Optional[bool]) -> HttpResponse
+                 create_stream_by_admins_only=REQ(validator=check_bool, default=None),
+                 allow_message_editing=REQ(validator=check_bool, default=None),
+                 message_edit_duration_seconds=REQ(validator=check_string, default=None)):
```

Comment:
The current thing has the minor benefit of: if someone puts an invalid value for message_edit_duration_seconds, and then unchecks allow_message_editing, we automatically set message_edit_duration_seconds to a reasonable value without throwing an error. Possibly not worth the added code complexity though.